client/connect-account-page/style.scss

```
@@ -125,4 +125,35 @@
 			}
 		}
 	}
+
+	.wcpay-connect-info-notice {
+		margin: 24px 0;
+		padding: 12px;
+		border-left: none;
+		font-size: 12px;
+		background-color: #fbf1fa;
+		color: $studio-gray-80;
+		display: flex;
+
+		img {
+			margin: $gap-small;
```

I think something like this will look better. Doesn't match the design entirely, but as we mentioned earlier, looks a bit better centered (IMO).

```suggestion
		margin: $gap-small 0 $gap-large;
		padding: $gap;
		font-size: 12px;
		background-color: #fbf1fa;
		color: $studio-gray-80;
		display: flex;

		img {
			margin:0 $gap 0 $gap-smallest;
```